scripts/docs/sources2html.pl

```diff
@@ -1232,14 +1228,22 @@ sub get_count {
     $count_display = $count;
     $bg_class = $colour_class{'lot_million'};
   }
-  # From 100 million
-  elsif ($count =~ /^(\d{3}\d*)\d{6}$/) {
+  # From 100 million to 999.9 million
+  elsif ($count =~ /^(\d{3})\d{6}$/) {
     my $number = $1;
     $count = "$number M";
     $count_label = "Over $number million $end_label";
     $count_display = $count;
     $bg_class = $colour_class{'hundred_million'};
   }
+  # From 1 billion to 9.9 billion
+  elsif ($count =~ /^(\d)(\d)\d{8}$/) {
```

Comment:
Could we reduce the redundancy here? It's the same function in two different scripts.

Reply:
Yes, not important for this PR. Thanks!